src/main/groovy/com/avioconsulting/mule/linter/model/Application.groovy

```diff
@@ -34,4 +34,14 @@ class Application {
         return file.exists()
     }
 
+    List<ProjectFile> getPropertyFiles() {
+        List<ProjectFile> files = []
+        applicationPath.eachDirRecurse { dir ->
+            dir.eachFileMatch(~/.*.properties/) { file ->
+                files.add(new ProjectFile(file))
+            }
+        }
```

Comment:
Like the pom file, this should get parsed and stored when loading an app, not each time we call the method.

Reply:
Also, what is the purpose of using the ProjectFile type, which really doesn't add anything to a File, if we're not extending it to be a PropertyFile?  Even if PropertyFile doesn't currently add anything, it most likely will need to in order to support getting a property value, counts and checking for encryption

Reply:
Also, should we actually be looking in the full application path, properties files need to be in src/main/resources or a sub-directory of it